### ISSUE TITLE
EWSF

### ISSUE THREAD

**Issue by Price-H16** [ApourtArtt/NostaleWidget#24]
When I replace EWSF and add EWSF.dll it just opens logger and that's all 
It's problem on my side or I do something wrong

**Comment by ApourtArtt** [comment]
Hello

This project is not using EWSF.dll (which is the original EWSF.EWS). Instead, it provides a way of replacing it.
Original EWSF exports two functions:
1) ShowNostaleSplash: used to display the splashscreen before initializing everything
2) FreeNostaleSplash: used to hide the splashscreen once everything is ready

You can override those functions there: [ShowNostaleSplash](https://github.com/ApourtArtt/NostaleWidget/blob/main/ClientModding/Example.cpp#L37) and [FreeNostaleSplash](https://github.com/ApourtArtt/NostaleWidget/blob/main/ClientModding/Example.cpp#L42)

It allows you to use EWSF.dll if you want ([see related project](https://github.com/xAfterLife/EWSF)) or even do something completely different (for example, show a better splashscreen with transparency, a GIF or whatever you want.

**Comment by Price-H16** [comment]
Thank you for reply but I'm stuck now on discord rpc logging in

**Comment by ApourtArtt** [comment]
Do you mean that the game window never appears ?

**Comment by Price-H16** [comment]
I mean that Discord rpc stuck on logging in even when I'm in game

**Comment by ApourtArtt** [comment]
There is already an issue about Discord RPC status: https://github.com/ApourtArtt/NostaleWidget/issues/25
I invite you to continue on it, I am closing this issue since EWSF is not related to Discord RPC, feel free to report any error on EWSF system if you encounter another issue, and thank you for your report!
